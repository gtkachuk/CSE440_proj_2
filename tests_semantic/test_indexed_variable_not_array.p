program testIndexedVariableNotArray;

class testIndexedVariableNotArray

BEGIN

   VAR 
      rows	: ARRAY[0 .. 9] OF boolean;
      tests	: boolean;
      moreTests	: boolean;

FUNCTION testIndexedVariableNotArray;
BEGIN	   
   tests := moreTests[5]
END

END
.

