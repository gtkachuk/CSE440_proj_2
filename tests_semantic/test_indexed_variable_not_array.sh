#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_indexed_variable_not_array.p
