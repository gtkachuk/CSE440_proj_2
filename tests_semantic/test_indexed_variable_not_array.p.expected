ERROR:15:Indexed variable "moreTests" is not an array
Errors detected. Exiting.
